pkg/oauth/handlers/default_auth_handler_test.go

```diff
@@ -182,10 +182,12 @@ func TestWithChallengeErrorsAndMergedSuccess(t *testing.T) {
 	if !handled {
 		t.Error("Expected handling.")
 	}
-	if !(reflect.DeepEqual(map[string][]string(responseRecorder.HeaderMap), expectedHeader1) || reflect.DeepEqual(map[string][]string(responseRecorder.HeaderMap), expectedHeader2)) {
-		t.Errorf("Expected %#v or %#v, got %#v.", expectedHeader1, expectedHeader2, responseRecorder.HeaderMap)
+
+	rec := responseRecorder.Result()
```

Comment:
what's rec? Use a variable name that is self-explanatory

Reply:
It is quite common to use short variable names in Go and rec being a common short name for a recorder.

https://github.com/golang/go/wiki/CodeReviewComments#variable-names=

Reply:
oh, yeah. this is true. :man_facepalming: